\chapter{Conclusion and Outlook}
\label{chap:6}
\clearpage
\raggedbottom

\section{Thesis summary and next steps}
  
Climate change is arguably one of the most urgent issues presently confronting the world. With no end in sight to the anthropogenic emission of carbon dioxide, expedited warming, sea level rise, and shifts in local and global weather patterns are likely. These changes, however, are not limited to terrestrial ecosystems, as coordinated changes in the temperature, carbonate chemistry, stratification, and nutrient environment of the ocean occur. These rapid changes to the Earth's ecosystem highlight the importance of a holistic understanding of the global carbon cycle, in which phytoplankton are key players. These climate-related changes to marine ecosystems will likely change the biogeography of phytoplankton, as organisms adapt to new environmental conditions. Predicting these changes to the phytoplankton distributions hinges upon understanding the response of phytoplankton both at the community and species-level. In this thesis, I presented new tools (Chapter \ref{chap:2}) and approaches (Chapter \ref{chap:3}-\ref{chap:5}) to survey, characterize, and quantify the response of phytoplankton to changes in their nutrient environment \textit{in situ}. 

\par
This thesis first introduces the development of a statistical approach to identify stably expressed genes for use both in qRT-PCR and metatranscriptomic studies from high-throughput sequencing datasets (Chapter \ref{chap:2}). Many of the ``housekeeping genes'' that are commonly used as references in qRT-PCR studies of phytoplankton were found to be variably expressed across the conditions used in this study. Such variability in a reference gene can drastically alter the gene expression patterns of the gene of interest. This approach broadens the pool of possible gene targets by facilitating the selection of genes without \textit{a priori} knowledge of function. Application of this technique to the metatranscriptomic study in chapter \ref{chap:3} identified unique stable reference genes for both of the dominant diatoms and facilitated the quantitative comparison of their transcriptional signals. 
\par
The field study in Narragansett Bay, presented in Chapter \ref{chap:3}, approached the problem of assessing and comparing diatom nutrient physiology in two ways. First, metabolic pathways with known relationships to N and P metabolism were tracked for both taxa. This approach suggested that the more dominant \textit{Skeletonema} was likely utilizing inorganic nutrients, where as the less abundant \textit{T. rotula} was relying upon organic compounds, like amino acids. This study, however, went further by combining field samples with incubation experiments. Using a Bayesian approach, these incubation experiments were used to 1) identify novel resource responsive gene targets and 2) contextualize expression signals to directly compare the physiology of organisms \textit{in situ}. This second step was accomplished by proportionalizing the signals in the field to the metabolic bounds on expression observed in the incubation experiments. These data demonstrated significant differences in the expression patterns of resource responsive genes, indicating that these two species are responding differently to the same nutrient environment. Narragansett Bay is an ideal model system for investigating the interplay between diversity and biogeochemistry as well as basic questions in ecology, such as the role of specialization in a dynamic ecosystem. Given the opportunity, I would want to sample this community throughout the year, combined with similar nutrient experiments. While this technique was designed to examine the response of phytoplankton to nutrient dynamics, it could quite easily be expanded to other systems or variables. 
\par
Chapters \ref{chap:4} and \ref{chap:5}, both set in the oligotrophic North Pacific Subtropical Gyre, look to the opposite ends of the diversity spectrum, assessing the differences in response to nutrient pulse amongst functional groups and amongst strains. By sampling and comparing the global gene expression of the eukaryotic community both \textit{in situ} and in simulated deep seawater (DSW) upwelling incubations, the specific drivers of production for taxonomic groups were identified (Chapter \ref{chap:4}). The divergence in the variable transcript allocation ratio (VTAR) between the diatoms and haptophytes was one of the more surprising outcomes of this study, as it suggests that the r- and K-strategies as defined by enzyme kinetics and growth rate appear to hold true at the level of the transcription. I am quite interested to see if this schism in diatom and haptophyte VTAR is consistent across other stimuli, timescales, and systems\footnote{A preliminary glance at the data from Narragansett Bay suggests that it might be.}. In the near future, I would like to replicate this study, but sample the community 24-hours post-DSW addition to assess the more immediate response of the community to nutrient addition, potentially removing the signals associated with shifts in the community structure. 
\par
The other rather perplexing outcome from this study was the lack of response seen in the dinoflagellates following DSW addition. In Chapter \ref{chap:4} I suggested that this might be due to the fact that dinoflagellates might have alternative trophic strategies or live symbiotically. However, I have a feeling that it is more of a general characteristic of dinoflagellate transcriptional regulation, or, rather, the lack there of. It is known that autotrophic dinoflagellates do respond at the protein and activity level to changes in their nutrient environment \citep{Dyhrman1999}. Though not discussed in this thesis, expression patterns of \textit{Prorocentrum} in the Narragansett Bay incubations suggest that this pattern of no regulation may hold true in coastal dinoflagellates as well. Together, these data seem to indicate that metabolic plasticity in the dinoflagellates is happening at the protein level. It remains to be seen if these general patterns are upheld in future transcriptomic and metatranscripomic studies. 
\par
Finally, Chapter \ref{chap:5} investigated the intersecting roles of metabolic plasticity and strain diversity in the response of the \textit{Emiliania huxleyi} species complex to changing nutrient environments. The co-existence of at least five strains was observed in the environment, as defined by the variable gene sets. Additionally, following the addition of N, the strain composition of the field appeared to be altered. These findings support the hypothesis of \citet{Read2013} that the variability in gene content as observed in the pan genome is present and modulated in the field. Additionally, metabolic plasticity, as defined by changes in transcript abundance, was observed, particularly in metabolic processes associated with N and P metabolism, as well as life stage and calcification. The fact that any genes were identified as significantly differentially regulated in this study is surprising, as the two ``biological replicates'' used were actually distinct incubation experiments that were performed approximately two weeks apart with different initial populations. This, combined with the fact that many of these gene targets were also identified as significantly differentially regulated in N and P limited cultures in proteomic work by \citet{McKew2015}, suggests that these responses to nutrient stress are highly conserved within the \textit{E. huxleyi} species consortium. This study, to me, highlights the importance of culturing diversity. In many ways, even with metatranscriptome assembly, we are limited by what we know; our ability to assign function or taxonomy to meta-omic data in the field hinges upon our databases. 
\par

Together these data provide new insights into the inner workings of the phytoplankton consortiums, illuminating potentially missed co-limitations, capabilities, and interactions. Yet, the ultimate fate of these data beyond my very narrow field, as of yet, remains uncertain. Currently, there is a disconnect between the type of data that biologists, such as myself, want to collect, and the type of data that ecosystem and ocean modelers want. The undertone of some of the chapters in this thesis (e.g. Chapter \ref{chap:4}) was that these findings might be integrated into biogeochemical models. While I do believe that progress is being made toward this ultimate goal\footnote{In particular, the Darwin Project \citep{Follows2007} and some interesting, as of yet unpublished, work by Victoria Coles are pushing the boundaries on this integration.}, there remain many hurdles \citep{Hood2007, Worden2015}. With continued collaboration and discourse between molecular ecologists and ecosystem, I am hopeful that a solid quantitative connection might be made. 
\clearpage
\section{A final thought}

\textit{``We cannot cheat on DNA. We cannot get round photosynthesis. We cannot say I am not going to give a damn about phytoplankton. All these tiny mechanisms provide the preconditions of our planetary life.'' }
\begin{flushright}-- Barbara Ward, \textit{Who Speaks for Earth?}\footnote{Barbara Ward et al. Edited by Maurice F. Strong. (1973) \textit{Who Speaks for Earth?} New York, Norton. }\end{flushright}

Barbara Ward, a British economist during the 20\textsuperscript{th} century, was an early, some might say before her time, proponent of sustainable development. She possessed, what I believe was, a very holistic view of the intersection of economic and environmental concerns, arguing strongly for humans to practice ``careful husbandry'' of the Earth. Though slightly crass, this quote, aside from hitting somewhat coincidentally upon the major objects of my thesis (phytoplankton, DNA...), neatly sums up a point that I have been struggling with during my PhD. For my research, I dove into the smallest detail, tracking the shifts in the RNA composition of a consortium of \textit{very} tiny organisms living within a \textit{very} small parcel of water in the middle of a \textit{very} large ocean. After analyzing these data, it was hard to rationalize the connection of these data to the larger scale dynamics of the ocean system. These findings at the sub-micro scale seemed too detached from the global system to have any real import. But, as Barbara Ward points out, these ``tiny mechanisms provide the preconditions of \textit{our} planetary life.'' 



%The ‘omic revolution has shifted the limiting step in understanding the microbial role in biogeochemical cycles away from difficulty in directly monitoring the metabolism of microbial communities to data integration. We must be able to quantitatively unify both physical and chemical environmental data with large ‘omic’ datasets of all varieties (from genomic to metabolomic) if we wish to address larger questions of ecology. This is a computational and quantitative problem that is of great interest to me as I believe it will allow us not only to better address overarching questions, but that cross use of these data will inform the analysis and interpretation of each individual data type. 




%\section{Conclusions and future directions}
%Capitalizing upon this revolution to address biogeochemical or ecological questions, however, is becoming increasingly difficult, as our ability to make measurements has surpassed our ability to analyze, visualize, and compare the data produced.



