%% This is an example first chapter.  You should put chapter/appendix that you
%% write into a separate file, and add a line \include{yourfilename} to
%% main.tex, where `yourfilename.tex' is the name of the chapter/appendix file.
%% You can process specific files by typing their names in at the 
%% \files=
%% prompt when you run the file main.tex through LaTeX.

\chapter{Nutrient pulses uniquely drive physiological ecology of cosmopolitan phytoplankton strains}
\raggedbottom
\begin{singlespace}
%Harriet Alexander$^{1,2}$, Bethany D. Jenkins$^{3,4}$, Tatiana A. Rynearon$^{3}$, Mak A. Saito$^{5}$, Melissa L. Mercier$^{3}$, Sonya T. Dyhrman$^{2}$\\

%$^{1}$ MIT-WHOI Joint Program in Oceanography/Applied Ocean Science and Engineering, Cambridge, MA 02139, USA\\
%$^2$ Biology Department, Woods Hole Oceanographic Institution, Woods Hole, MA 02543, USA\\
%$^3$ Graduate School of Oceanography, University of Rhode Island, Narragansett, RI 02882, USA\\
%$^4$ Department of Cell and Molecular Biology, University of Rhode Island, Kingston, RI 02881, USA\\
%$^5$ Department of Marine Chemistry and Geochemistry, Woods Hole Oceanographic Institution, Woods Hole, MA 02543, USA\\
%\\
%Previously published: Alexander, H., Jenkins, B.D., Rynearson, T.A., Saito, M.A., Mercier, M.L., and Dyhrman, S.T. (2012). Identifying reference genes with stable expression from high throughput sequence data. \emph{Front. Microbiol.} 3, 385.
\end{singlespace}

\section{Abstract}
Phytoplankton are well tuned to respond to changing environments, this may happen at the community-level with species or functional group succession, at the species-level through shifts in strain composition, or at the strain-level through alterations to phenotype. Community-level shifts have been well described; however, strain or phenotypic shifts have been more difficult to diagnose in the field. Here, we examined the intersecting roles of metabolic plasticity and strain diversity in the response of natural populations of the common cocolithophore \textit{Emiliania huxleyi} to shifting nutrient regimes in the North Pacific Subtropical Gyre (NPSG). Using a metatranscriptomic approach and leveraging the genome of strain CCMP 1516 and the transcriptomes of four strains of \textit{E. huxleyi} sequenced through the Marine Microbial Eukaryote Transcriptome Sequencing Project (MMETSP), we combined field observations and microcosm studies to track compositional and metabolic responses to nutrient pulses. The \textit{E. huxleyi} transcript pool in the NPSG was dominated by the genome strain. Following the addition of nitrogen, there was a reduction in genome strain transcripts and an increase in transcripts identified as “core,” or common to all strains used in this study. These data suggest that the nutrient pulse caused a shift in the strain composition to a strain not included in our reference set that likely exists at fairly low levels in the NPSG but is adept at responding to nutrient pulses. In addition to the variations in strain diversity observed following nutrient addition, significant changes in transcript abundance were observed and suggest that nitrogen addition induced shifts in the ploidy of the population and changes in nutrient physiology and calcification state. Together, these data underscore the ecological import of the pan-genome of \textit{E. huxleyi}, suggesting that genetic variability within the species complex may be at the heart of its success in a wide variety of marine environments.  
\section{Introduction}

\section{Materials and Methods}
\section{Results and Discussion}
