

\chapter{Physiological response and strain variation of the \textsl{\textsc{Emiliania huxleyi}} species complex under changing nutrient environments}
\raggedbottom

\clearpage


\section{Abstract}
Phytoplankton are well tuned to respond to changing environments, this may happen at the community-level with functional group succession, at the species-level through shifts in strain composition, or at the strain-level through alterations to phenotype. Community-level shifts have been well described; however, strain or phenotypic shifts have been more difficult to identity and describe in the field. Here, we examined the intersecting roles of metabolic plasticity and strain diversity in the response of natural populations of the biogeochemically significant cocolithophore Emiliania huxleyi to shifting nutrient regimes in the North Pacific Subtropical Gyre (NPSG). Using a metatranscriptomic approach, field observations were paired with microcosm studies to track the compositional and metabolic responses to shifts in the geochemical environment. The transcriptomes and genome of five strains were clustered based on protein homology to identify the ‘core’ set of genes common across strain, as well as sets of genes unique to each strain. These strain-specific gene sets were used to track strain composition in the field and microcosms. The strain composition of the in situ samples varied little over the sampling period, with transcripts specific to strains CCMP1516, CCMP370 and PLYM219 the most abundant.  Following the addition of nitrogen, however, transcripts specific to strains CCMP374 and CCMP379 exhibited dramatic increases. In addition to the variations in strain diversity observed following nutrient addition, significant changes in transcript abundance were observed for gene pathways involved in nitrogen, and phosphorus metabolism.  The data suggested that nitrogen is a major driver of the physiological ecology of E. huxleyi in this system, and nitrogen supply may be linked to shifts in the ploidy of the population and changes in nutrient physiology and calcification state. Together, these data underscore the ecological importance of the “pan genome” of E. huxleyi, suggesting that genetic variability within the species complex may be at the heart of its success in a wide variety of marine environments. 
\section{Introduction}

\section{Materials and Methods}
\subsection{Sample collection and shipboard nutrient incubation experiments}
Seawater for the in situ eukaryote community mRNA analysis was collected at the HOT, Station ALOHA (22˚45’ N, 158˚00’ W) from a depth of 25 m at 1400 hrs (local time) on six occasions during the summer of 2012 (S1: 6 August, S2: 12 August, S3: 24 August, S4: 30 August, S5: 2 September, S6: 5 September) using a Eulerian sampling scheme as part of the HOE-DYLAN research expedition as per Alexander et al. (2015b). Water was collected in acid-washed 20-L carboys and approximately 60 L of seawater was prescreened through 200-µm mesh and then filtered onto polycarbonate filters (5.0 µm pore size, 47 mm, Whatman) by way of peristaltic pump. Filters were changed every 20 minutes or when flow rate decreased. Filters were placed in cryovials and stored in liquid nitrogen until mRNA extraction. The total length of filtration time did not exceed 3 hours. 
In conjunction with these field-based surveys, two factorial nutrient amendment incubation experiments focused on the macronutrients N and P were performed with natural communities (T0 of E1 was T0 of E2 was S4) (STable 1). Incubations were modeled after a simulated 10\% deep seawater (DSW) upwelling as described in Alexander et al. (2015b) and designed to tease apart the potential nutritional components of DSW upwelling. The concentration of iron was modeled after Marchetti et al. (2012) and vitamin B12 was modeled after Bertrand (2007). Triplicate 20-L carboys of each treatment were incubated at 30\% surface light-levels using on-deck incubators for 7 days and processed as described above, on the final day at 1400 hrs (local time). Nutrient concentrations for phosphate [PO4], nitrate and nitrite [NO2+NO3] were measured by filtering 125 mL of seawater through a 0.2-μm, 47-mm polycarbonate filter, and stored frozen (−20 °C) in acid washed bottles until analysis at the Chesapeake Bay Lab at the University of Maryland according to the facility's protocols. Samples for alkaline phosphatase activity (APA) were collected by filtering 250-ml of whole seawater onto polycarbonate filters (0.2 µm pore size, 47 mm, Whatman) and frozen at -20oC. These filters were then resuspended in artificial seawater and assayed for APA fluorometrically using the fluorogenic phosphatase substrate 6,8-difluoro-4-methylumbelliferyl phosphate (diMUF-P, Molecular probes) following established field protocols (Dyhrman and Ruttenberg 2006). Chlorophyll a was measured on whole water samples collected onto GF/F filters (25 mm, Whatman) using a 90\% acetone extraction and assayed by fluorescence using the AquaFluor Turner TD700 (Parsons et al. 1984).
\subsection{RNA extraction and sequencing}
RNA was extracted from individual filters with the RNeasy Mini Kit (Qiagen), following a modified version of the yeast protocol. Briefly, lysis buffer and RNA-clean zirconia/silica beads was added to the filter and samples were vortexed for 1 minute, placed on ice for 30 seconds, and then vortexed again for 1 minute. Samples were then processed following the yeast protocol. The resulting RNA was eluted in water and then treated for possible DNA contamination using TURBO DNA-free Kit (Ambion) following the Rigorous DNase protocol. RNA from individual filters was then pooled by sample, using the RNA Cleanup Protocol from the RNeasy Mini Kit (Qiagen). The resulting RNA sample thus represented approximately 56 L of total seawater for the in situ sample. Filters were pooled across like triplicate bottles by treatment, totaling 56 L from each of the incubation treatments. The total RNA sample was then enriched for eukaryotic mRNA through a poly-A pull down. The resulting enriched mRNA sample then went through library preparation with the Illumina TruSeq mRNA Prep Kit (Illumina). Libraries were sequenced with the Illumina HiSeq2000 at Columbia Genome Center (New York, NY). Each sample was sequenced to produce a targeted 60 million, 100 base pair, paired end reads. Raw sequence data quality was visualized using FastQC and then cleaned and trimmed using Trimmomatic v 0.27 (paired end mode; 6-base pair wide sliding window for quality below 20; minimum length 25 base pair). 
\subsection{Community and strain specific mapping and expression analysis}
Transcriptome sequences and annotations generated through the Marine Microbial Eukaryote Transcriptome Sequencing Project (MMETSP) that were made public as of 17 March 2014 were collected and treated as per Alexander et al. (2015b) to track species composition of the metatranscriptomes. Due to the large size of the resulting MMETSP database, trimmed reads from the metatranscriptome were mapped to the MMETSP using the Burrows-Wheeler Aligner (Li and Durbin 2010) (BWA-mem, parameters: -k 10 -aM) and then counted using the HTSeq 0.6.1 package (Anders et al. 2014). 
The combined transcriptomes (as assembled from the NCGR on 4 September 2013) from unialgal cultures of Emiliania huxleyi strains CCMP374 (MMETSP1006-MMETSP1009), CCMP379 (MMETSP0994-MMETSP0997), CCMP370 (MMETSP1154-MMETSP1157), and PLYM219 (MMETSP1150-MMETSP1153). All transcriptome assemblies used are available through the iMicrobe data commons. Additionally, the predicted transcripts from the E. huxleyi genome, strain CCMP1516, were used. All transcriptomes were trimmed based on predicted peptide length, requiring sequences be longer than 70 amino acids. The resulting set of genes was considered for subsequent analyses. Peptide sequences were clustered into gene clusters with orthoMCL (citation), using standard parameters: BLASTP with an e-value cutoff of 1e-5, and an inflation value (-I) of 1.5. Initially, the transcripts unique to CCMP1516, here surveyed using the predicted transcripts from the genome, were the most dominant of the subsets of genes in these analyses, representing ~50\% of the E. huxleyi reads in the field (Supplemental Figure Core Changes). Closer inspection demonstrated that many of the most highly represented genes identified as unique to CCMP1516 were associated with metabolic stasis or senescence (e.g. OG1_5_1124, a group of homologous proteins in the E. huxleyi genome such as JGI # 413698 annotated as putative senescence-related proteins and highly expressed in all field samples). Many of the proteins in the unique set of CCMP1516 were identified as “core” amongst the 13 strains surveyed by Read et al. (2013), yet were absent in some or all of the transcriptomes of the four strains in this study. This absence likely is related to the fact that these strains were largely sampled under exponential growth conditions, limiting the expression of genes that might be associated with stressors or stasis. The lack of ‘core’ gene representation in some of these transcriptomes underscores the importance of growth condition in transcriptome complete
Using this clustering framework, field and incubation samples were mapped to the data set using RSEM, a software package designed to estimate gene and isoform expression values from RNA-seq data. Here we define orthologous groups as genes and individual transcripts (from any strain) as isoforms. Data were mapped using RSEM version 1.2.20 (parameters: --paired-end -p8 -bowtie2 -bowtie2-mismatch-rate 0.2). A note: RSEM is not yet able to deal with gapped mapping, such as enabled by bwa, which was used for the community-level mapping due to database size constraints. Taking a conservative approach, the RNA abundances from like treatments (each consisting of pooled triplicate bottles), which were run with different communities from separate water masses more than two weeks apart, were considered to be biological replicates for differential abundance analysis. These analyses were run with ll using default parameters to calculate dispersion and assess differential abundance of both individual transcripts and orthologous groups of each of the amended incubations compared to the no-addition control. Looking to previous literature, genes thought to be associated with nitrogen and phosphorus metabolism (Dyhrman et al. 2006; Rokitta et al. 2014; McKew et al. 2015), calcification and ploidy state (von Dassow et al. 2009; Mackinder et al. 2011; Frada et al. 2012), were compared against the translated proteins comprising the orthologous groups used in this study (tblastn with an e-value cutoff of 1e-20).

\section{Results and Discussion}
